com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LayerListUIBlock.cs

```diff
@@ -10,9 +10,12 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    class LayerListUIBlock : MaterialUIBlock
+    /// <summary>
+    /// Material Layer list ui block. Used in the Layered Lit shader.
```

Comment:
The UI block that represents a material layer list. The Layered Lit shader uses this UI block.